dev/doc/archive/extensions.txt

```diff
@@ -10,9 +10,9 @@ Exemple de l'ajout de l'entrée "clause":
   ce nouveau ClauseArgType
 
 - utiliser le rawwit_clause pour définir une entrée clause du bon
-  type et du bon nom dans le module Tactic de pcoq.ml4
+  type et du bon nom dans le module Tactic de pcoq.mlg
```

Comment:
I think probably the .ml4 in archive.txt should stay as they are, as they are historical references, not meant to describe the current system.